aea/cli/interact.py

```diff
@@ -79,7 +79,7 @@ def _run_interaction_channel():
 
     except KeyboardInterrupt:
         click.echo("Interaction interrupted!")
-    except Exception as e:  # pylint: disable=broad-except # pragma: no cover
+    except BaseException as e:  # pylint: disable=broad-except # pragma: no cover
```

Comment:
can remove pylint disable in this case?

Reply:
no, we can's. it still broad exception match